content/en/docs/installation/operator.md

```diff
@@ -1,59 +1,22 @@
-
-Operator has two distinct modes you can install and use:
-
-- **Basic**: Operator in basic mode installs Spinnaker into a single namespace without `ValidatingAdmissionWebhook` for doing preflight checks.
-- **Cluster**: Operator in cluster mode installs Spinnaker across namespaces with `ValidatingAdmissionWebhook` for doing preflight checks. This mode requires a `ClusterRole`.
-
-If you want to get started quickly, install Operator and Spinnaker by running the following commands:
-
-```
-# Pick a release from https://github.com/armory-io/spinnaker-operator/releases
-mkdir -p spinnaker-operator && cd spinnaker-operator
-bash -c 'curl -L https://github.com/armory-io/spinnaker-operator/releases/latest/download/manifests.tgz | tar -xz'
-
-# Install or update CRDs cluster wide
-kubectl apply -f deploy/crds/
-
-# Install Operator in namespace spinnaker-operator. If you want a different namespace, see the below documentaiton.
-kubectl create ns spinnaker-operator
-kubectl -n spinnaker-operator apply -f deploy/operator/cluster
-
-# Install Spinnaker in the "spinnaker" namespace
-kubectl create ns spinnaker
-kubectl -n spinnaker apply -f deploy/spinnaker/basic
-
-# Watch the install progress and check out the pods being created too!
-kubectl -n spinnaker get spinsvc spinnaker -w
-```
-
-The rest of this page describes how to modify some of the default configurations within Operator to suit your needs.
-
-## Benefits of Operator
-
-- Stop using Halyard commands: just `kubectl apply` your Spinnaker configuration. This includes support for local files.
-- Expose Spinnaker to the outside world (with `LoadBalancer`). You can still disable that behavior if you prefer to manage ingresses and load balancers yourself.
-- Deploy any version of Spinnaker. Operator is not tied to a particular version of Spinnaker.
+- Manage Spinnaker with `kubectl` like other applications.
+- Expose Spinnaker via `LoadBalancer` or `Ingress` (optional)
 - Keep secrets separate from your config. Store your config in `git` and have an easy Gitops workflow.
 - Validate your configuration before applying it (with webhook validation).
-- Store Spinnaker secrets in [Kubernetes secrets](https://github.com/armory/spinnaker-operator/blob/release-0.3.x/doc/managing-spinnaker.md#secrets-in-kubernetes-secrets).
-- Patch versions, accounts or any setting with `kustomize`.
-- Monitor the health of Spinnaker via `kubectl`.
+- Store Spinnaker secrets in [Kubernetes secrets](https://github.com/armory/spinnaker-operator/blob/master/doc/managing-spinnaker.md#secrets-in-kubernetes-secrets).
+- Gain total control over Spinnaker manifests with [`kustomize` style patching](https://github.com/armory/spinnaker-operator/blob/master/doc/options.md#speckustomize)
 - Define Kubernetes accounts in `SpinnakerAccount` objects and store kubeconfig inline, in Kubernetes secrets, in s3, or GCS **(Experimental)**.
+- Deploy Spinnaker in an Istio controlled cluster **(Experimental)**
```

Comment:
Would this list be better served in the product compatibility matrix? i.e. could we just link to our intro blog post here instead of listing all this out?